zircon-object/src/task/exception.rs

```diff
@@ -105,11 +109,11 @@ impl Exceptionate {
 }
 
 #[repr(C)]
-pub struct ExceptionInfo {
-    pub pid: KoID,
-    pub tid: KoID,
-    pub type_: ExceptionType,
-    pub padding: u32,
+struct ExceptionInfo {
+    tid: KoID,
+    pid: KoID,
```

Comment:
Wrong order. Maybe here is why the core-test failed.

Reply:
Oops, I missed the change when merging :(